tools/CreateDebugNuget.ps1

```diff
@@ -42,6 +42,7 @@ Copy-Item -Path $buildDir -Destination $tempBuildDir -Recurse
 #source nuspec file paths
 $sourceRDNuspec = Join-Path $sourceDir "ReactiveDomain.Debug.nuspec"
 $sourceRDPolicyNuspec = Join-Path $sourceDir "ReactiveDomain.Policy.Debug.nuspec"
+$sourceRDPolicyTargets = Join-Path $sourceDir "ReactiveDomain.Policy.targets"
```

Comment:
Should this and the related changes in this file also be made to `CreateNuget.ps1`?